nbconvert/exporters/slides.py

```diff
@@ -103,6 +103,21 @@ def _reveal_url_prefix_default(self):
         """
     ).tag(config=True)
 
+    reveal_transition = Unicode('linear',
+        help="""
+        Name of the reveal.js transition to use.
+
+        The list of themes that ship by default with reveal.js are:
+        default, cube, page, concave, zoom, none.
+        """
+    ).tag(config=True)
+
+    reveal_scroll = Bool(False,
+        help="""
+        Whether to enable/disable scrolling per slide basis
```

Comment:
>If True, enable scrolling within each slide

I agree.

>If this is disabled, does scrolling move between slides? Or just do nothing?

Just do nothing... scrolling to move between slides can be configured with other reveal.js parameter (`mousewheel` IIRC), but we don't surface that in our template.